framework/packages/abstract-client/src/account.rs

```diff
@@ -176,32 +236,43 @@ impl<Chain: CwEnv> Account<Chain> {
         Ok(coins[0].amount)
     }
 
+    /// Query account balances of all denoms
     pub fn query_balances(&self) -> AbstractClientResult<Vec<Coin>> {
         self.environment()
             .balance(self.proxy()?, None)
             .map_err(Into::into)
             .map_err(Into::into)
     }
 
-    // TODO: remove `get_account` prefix
-    // Getters are not prefixed with `get_` in rust.
-    pub fn get_account_info(&self) -> AbstractClientResult<AccountInfo<Addr>> {
+    /// Query account info
+    pub fn info(&self) -> AbstractClientResult<AccountInfo<Addr>> {
         let info_response: InfoResponse = self.abstr_account.manager.info()?;
         Ok(info_response.info)
     }
 
-    // Install an application on the account
-    // creates a new sub-account and installs the application on it.
-    pub fn install_app<
-        M: ContractInstance<Chain> + InstallConfig + From<Contract<Chain>> + Clone,
-    >(
+    /// Install an application on the account.
+    /// This creates a new sub-account and installs the application on it.
+    pub fn install_app<M: ContractInstance<Chain> + InstallConfig + From<Contract<Chain>>>(
         &self,
         configuration: &M::InitMsg,
         funds: &[Coin],
     ) -> AbstractClientResult<Application<Chain, M>> {
         self.install_app_internal(vec![M::install_config(configuration)?], funds)
     }
 
+    pub fn install_adapter<
```

Comment:
This method doesn't really make sense IMO. 

Adapters should get installed on the current account. Not a new sub-account.

Reply:
Hm, that's an interesting point! Should we have a way to install apps on current account as well?

Reply:
Or, what about this: 

We add a `install_on_sub_account` boolean attribute to the `Account` that can be set in the builder and defaults to `true`. That way it's a global setting and you can do whatever. 

Want to install adapters on sub-accounts? cool. 
Want to install apps on current account? done.
etc

Reply:
We have multiple ways of obtaining account beyond builder, do we want to have ability to set it on `Account` object as well? And I think depending on a context we might want to use different defaults values
- `from_namespace` -> true
-  `install_*` -> if installed on sub account false(so you don't create another sub-account on top of it accidentally), else true
- `get_last_account` - true

Reply:
> We have multiple ways of obtaining account beyond builder, do we want to have ability to set it on `Account` object as well? And I think depending on a context we might want to use different defaults values
> 
> * `from_namespace` -> true
> * `install_*` -> if installed on sub account false(so you don't create another sub-account on top of it accidentally), else clone Account object
> * `get_last_account` - true

If we enable fetch_from_namespace by default then we don't need the `from_namespace` anymore.